frontend/src/components/StakeCard/metricCard.css

```diff
@@ -1,12 +1,20 @@
 .stake-card {
-  width: 309px;
+  /* width: 309px; */
```

Comment:
why did you remove this?

Reply:
There was no difference in having it there or removing it. That's why i removed it